src/components/steps/general-info/Funders.tsx

```diff
@@ -72,8 +75,17 @@ const Funders: React.FC<FundersProps> = ({ project, onChange }) => {
                         ordered={false}
                         height={300}
                         onChange={(selected: string[]) => {
+                            setSelectedFunders(selected);
                             onUpdateField("funders", funderOptions, selected);
-                            setDialogOpen(true);
+                            if (selectedFunders.length < selected.length) {
+                                setDialogOpen(true);
+                            } else if (selectedFunders.length > selected.length) {
+                                snackbar.success(
```

Comment:
As I understand the task, here we need to update the additional designation. Note: I guess you'll reuse the `newAdditional` code, abstract it not to duplicate code. Also, it's a good moment to change pop by _.last